src/Native/Runtime/Volatile.h

```diff
@@ -39,22 +39,8 @@ inline
 T VolatileLoad(T const * pt)
 {
 #ifndef DACCESS_COMPILE
-#if defined(_ARM64_) && defined(__GNUC__)
```

Comment:
I assume that this is fixing errors about initializing of type `int *` to `volatile int *`. Could you please fix it same way it is fixed in CoreCLR? https://github.com/dotnet/coreclr/blob/master/src/inc/volatile.h#L141